fastlane/Fastfile

```diff
@@ -0,0 +1,83 @@
+# Customise this file, documentation can be found here:
+# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
+# All available actions: https://docs.fastlane.tools/actions
+# can also be listed using the `fastlane actions` command
+# Change the syntax highlighting to Ruby
+# All lines starting with a # are ignored when running `fastlane`
+# If you want to automatically update fastlane if a new version is available:
+# update_fastlane
+# This is the minimum version number required.
+# Update this, if you use features of a newer version
+fastlane_version "2.120.0"
+default_platform :ios
+# Set default target for get_version_number()
+ENV["FL_VERSION_NUMBER_TARGET"] = "Examples"
+SPACESHIP_SKIP_2FA_UPGRADE=1
```

Comment:
Should it be environment variable or fastlane is smart enough to process this expression?

Reply:
From my testing, fastlane was smart enough to process the expression as is.

Reply:
Correction: I changed it to an environment variable.